command/root.go

```diff
@@ -351,3 +352,37 @@ func determineEditor(cmd *cobra.Command) (string, error) {
 
 	return editorCommand, nil
 }
+
+func ExpandAlias(args []string) ([]string, error) {
+	// TODO i don't love that context is being init'd twice. My reading of the code is that it is
+	// idempotent and okay but I don't know if that's ok to depend on.
+	if len(args) == 1 {
+		// the command is lacking a subcommand
+		return []string{}, nil
+	}
+
+	ctx := initContext()
+	cfg, err := ctx.Config()
+	if err != nil {
+		return []string{}, err
+	}
+	aliases, err := cfg.Aliases()
+	if err != nil {
+		return []string{}, err
+	}
+
+	if aliases.Exists(args[1]) {
+		expansion := aliases.Get(args[1])
+		if strings.Contains(expansion, "$") {
+			for i, a := range args[2:] {
+				expansion = strings.Replace(expansion, fmt.Sprintf("$%d", i+1), a, 1)
+			}
+			if strings.Contains(expansion, "$") {
+				return []string{}, fmt.Errorf("not enough arguments for alias: %s", expansion)
+			}
+		}
+		return shlex.Split(expansion)
```

Comment:
According to examples in the command documentation, any extra arguments should be forwarded after the alias expansion:
```sh
gh alias set pv 'pr view'
# gh pv -w 123 -> gh pr view -w 123
```
but that does not seem to happen here. Unless the alias expansion claimed some extra values through `$<N>` positional placeholders, any extra arguments will currently get ignored.